Team-Ada-Lovelace/Capstone-PartyPal
# Summary component

# Context
Needed an event summary component 

# What this PR does?
Shows an event summary component when front-end URL matches /eventSummary.

# How to test?
Go into this branch, run the server, and change the URL to /eventSummary should see GH Graduation event details.
